src/server/frontend_xwayland/xcb_connection.h

```diff
@@ -78,7 +78,6 @@ class XCBConnection
 
     private:
         Atom(Atom&) = delete;
-        Atom(Atom&&) = delete;
```

Comment:
Any particular reason you've added (or at least, un-removed) the move constructor here?

Reply:
According to [this SO answer](https://stackoverflow.com/a/23771245) an explicitly deleted copy constructor counts as a user-declared copy constructor. This is supported by [this Compiler Explorer code](https://gcc.godbolt.org/z/GU8cSd) (note how it starts compiling when you comment out the deleted copy constructor).